src/codegen/riscv64/assembler-riscv64.h

```diff
@@ -641,6 +641,7 @@ class V8_EXPORT_PRIVATE Assembler : public AssemblerBase {
   void jal(int32_t imm21) { jal(ToRegister(1), imm21); }
   inline void jal(Label* L) { jal(jump_offset(L)); }
   void jr(Register rs) { jalr(zero_reg, rs, 0); }
+  void jalr(Register rs, int32_t imm12) { jalr(zero_reg, rs, imm12); }
```

Comment:
Jalr means Jump and Link Register, but the implementation use zero_reg, and the behavior is just jump.  Change the function name "jalr"  to "jr"?

Reply:
yes, i fix it now.